### PR TITLE
ZE ReadMe update

### PR DESCRIPTION
**Pull request by IgorCATech**
+ the rename + copy features 
+ user name and pw while creating a new profile are now optional (reviewers, please, make sure the formulation is correct (line 46).
+ Data Set rename and copy gifs added
+ "Prerequisites" section renamed to "Software requirements", to align with Zowe docs (good catch @BrandonJenkins14)
  - Node.js v8.0 or later added as a prepreq

### CODE REVIEW

**Comment by crawr** [comment]
Hi @IgorCATech , the copy data sets feature was merged to master yesterday. It would be good if this could be added in the sample use cases section as well.

**Comment by IgorCATech** [comment]
@jellypuno @crawr Please, review the copy USS file use case text. Not sure if the procedure is 100% correctly described.

File: README.md [hunk 1]
```diff
@@ -95,8 +99,10 @@ Review the following use cases to understand how to use Zowe Explorer.
 You can use the following functionalities when interacting with data set:
 
 * **View data sets and use multiple filters**: You can view multiple data sets simultaneously and apply filters to show specified data sets.
+* **Rename data sets**: You can rename specified data sets.
+* **Copy data sets**: You can copy a specified member inside a dat set.
```
Comment (crawr) [inline README.md]: Copy works for data sets and data set members

File: README.md [hunk 2]
```diff
@@ -226,6 +256,22 @@ You can use the following functionalities when interacting with USS files:
...
 
+#### Rename USS files
+
+1. Navigate to the explorer tree.
+2. Open the **USS** bar.
+3. Select a USS file you want to rename.
+4. Right-click the USS file and select the **Rename USS file** option.
+5. Change the name of the USS file.
+
+#### Copy USS files
```
Comment (crawr) [inline README.md]: Copy functionality for USS files has not yet been implemented for this next release

File: README.md (hunk 1 above)
Comment (jalel01) [reply README.md]: Typo 'dat set'

Comment (IgorCATech) [reply README.md]: @jalel01 nice catch! thanks.
@crawr Do you mean that it's possible to copy an entire data set?

Comment (jellypuno) [reply README.md]: @IgorCATech yep. you can copy datasets. I tried copying Sequential to Sequential and Sequential to PDS Member v.v.

Comment (IgorCATech) [reply README.md]: oh ok... so not all types of data sets are possible to be copied. Thanks for the answer.

**Pull request merged**